drones/main.py

```diff
@@ -21,5 +22,22 @@ def setup_log():
 
 if __name__ == "__main__":
     setup_log()
-    c.connect()  # Sample entrypoint call
-    # c.__track_errors()    # Sample invalid call - function inaccessible
+
+    while True:
+
+        print("Input command: ")

```

Comment:
Use logger object instance of `print`.

Reply:
Do you mean creating another logger object instance for printing?
Logging documentation suggests using print() (https://docs.python.org/3/howto/logging.html)

![image](https://user-images.githubusercontent.com/48533475/112730927-996c1200-8f34-11eb-8958-26eae67d6e91.png)

Reply:
Good catch, I didn't notice it was a prompt. In this case, I would suggest moving it to `input` e.g. `command_to_send = input("Input command:\n")` (or format it however you want).